TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs

```diff
@@ -110,7 +114,8 @@ public int GetDigestSize()
 
         public void Reset()
         {
-            _message = new byte[0];
+            Array.Clear(_message, 0, _message.Length);
```

Comment:
You can use **CryptographicOperations.ZeroMemory(Span<Byte>)** Method for net6.0 and .NET Standard 2.1 
`#if NETCOREAPP2_1_OR_GREATER || NETSTANDARD2_1_OR_GREATER`

and add ` [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]` attribute to **Reset** method

Reply:
it's enough to use one thing or  CryptographicOperations.ZeroMemory(Span) or  [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]
but I recommend using CryptographicOperations.ZeroMemory(Span)

Reply:
CryptographicOperations.ZeroMemory(Span) unsupported by .NET Standard 2.0

Reply:
I think the best thing is to make our own method